SPECS/el9/gpsbabel.spec

```diff
@@ -42,9 +42,11 @@ Qt GUI interface for GPSBabel
 
 
 %build
+# LTO appears to cause some issues.
+# https://bugzilla.redhat.com/show_bug.cgi?id=2065758
+%define _lto_cflags %{nil}
 %cmake \
   -DGPSBABEL_WITH_ZLIB=pkgconfig \
-  -DGPS_BABEL_WITH_SHAPE_LIB=pkgconfig \
```

Comment:
This variable is not used:
https://github.com/radiant-maxar/geoint-deps/actions/runs/3884999029/jobs/6628981925#step:3:315